Source/VisualRecognitionV3/VisualRecognition+UIImage.swift

```diff
@@ -134,6 +134,42 @@ extension VisualRecognition {
         self.detectFaces(imagesFile: file, url: nil, failure: failureWithDelete, success: successWithDelete)
     }
 
+    /**
+     Classify an image using a Core ML model from the local filesystem.
+
+     - parameter image: The image to classify.
+     - parameter classifierIDs: A list of the classifier ids to use. "default" is the id of the
+     built-in classifier.
+     - parameter threshold: The minimum score a class must have to be displayed in the response.
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed with the image classifications.
+     */
+    @available(iOS 11.0, tvOS 11.0, watchOS 4.0, *)
+    public func classifyWithLocalModel(
+        image: UIImage,
+        classifierIDs: [String] = ["default"],
+        threshold: Double? = nil,
+        failure: ((Error) -> Void)? = nil,
+        success: @escaping (ClassifiedImages) -> Void)
+    {
+        // save image to disk
+        let file: URL
+        do {
+            file = try saveToDisk(image: image)
```

Comment:
@mediumTaj That's correct, you can get UIImage directly from the camera and from the image picker.

Reply:
Good point @mediumTaj. Since the other `classifyWithLocalModel()` method converts a file to `Data`, we could likewise turn the `UIImage` in this method to `Data`. Then both methods call a common method that accepts `Data` for the image parameter.

Reply:
Agreed.  I will fix.

Reply:
Yep - avoiding the write to disk is important for the `UIImage` method

Reply:
If this is going to be a breaking change to the signature of this function, should we wait until the release of the next major version?

In the meantime, we could add a cross-platform extension like this:
https://gist.github.com/JohnSundell/05f837a3f901630e65e3652945424ba5

Reply:
This is not a breaking change.